.github/workflows/kind.yaml

```diff
@@ -86,16 +102,37 @@ jobs:
           cilium uninstall --wait
 
       - name: Install Cilium with IPsec Encryption
+        if: ${{ matrix.mode == 'classic' }}
         run: |
           cilium install \
           --version=${{ env.cilium_version}} \
           --encryption=ipsec \
           --helm-set kubeProxyReplacement=disabled
 
+      - name: Install Cilium with IPsec Encryption
+        if: ${{ matrix.mode == 'helm' }}
+        run: |
+          kubectl create -n kube-system secret generic cilium-ipsec-keys \
```

Comment:
Can't we still have this handled by the CLI even in Helm mode?

Reply:
yeah we could. i do want to have a discussion first though and come up with a general guideline on how cilium-cli should handle resources that are not part of the helm chart.

on the one hand, these flags are very convenient. on the other hand, i think it's a very slippery slope. we might end up adding more and more logic to cilium-cli without carefully considering:

- what resources should be managed by cilium, and what resources should be managed by the user.
- if cilium should be managing the resource, why it's not part of the helm chart.